static/src/javascripts-legacy/projects/common/modules/experiments/ab-test-clash.js

```diff
@@ -21,15 +21,31 @@ define([
         name: 'ContributionsEpicAskFourEarning',
         variants: ['control']
     };
+    var GuardianTodaySignupMessaging = {
+        name: 'GuardianTodaySignupMessaging',
+        variants: ['message-a', 'message-b', 'message-c']
+    };
+
+    var contributionsTests = [
+        ContributionsEpicAlwaysAskStrategy,
+        ContributionsEpicBrexit,
+        ContributionsEpicAskFourStagger,
+        ContributionsEpicAskFourEarning
+    ];
+
+    var emailTests = [
+        GuardianTodaySignupMessaging
+    ];
 
-    function userIsInAClashingAbTest() {
-        var clashingTests = [
-            ContributionsEpicAlwaysAskStrategy,
-            ContributionsEpicBrexit,
-            ContributionsEpicAskFourStagger,
-            ContributionsEpicAskFourEarning
-        ];
-        return _testABClash(ab.isInVariant, clashingTests);
+    var nonEmailClashingTests = contributionsTests;
+
+    var clashingTests = contributionsTests.concat(emailTests);
+
+    function userIsInAClashingAbTest(tests) {
+        if (tests == null) {
```

Comment:
`if (!tests)` is generally a more idomatic way to do this - it will check for `0, false, ''` and `null`